docs/tutorials/nb_shap_feature_elimination.ipynb

```diff
@@ -8,7 +8,7 @@
     "\n",
     "[![open in colab](https://colab.research.google.com/assets/colab-badge.svg)](https://colab.research.google.com/github/ing-bank/probatus/blob/master/docs/tutorials/nb_shap_feature_elimination.ipynb)\n",
     "\n",
-    "Recursive Feature Elimination allows to efficiently reduce the number of features in your dataset, without losing the predictive power of the model. `probatus` implements the following feature elimination routine for **tree-based models**:\n",
+    "Recursive Feature Elimination allows to efficiently reduce the number of features in your dataset, without losing the predictive power of the model. `probatus` implements the following feature elimination routine for **tree-based,linear models**:\n",
```

Comment:
Isn't it for any provided model?

Good pick are often tree-based or linear, but can be also other.

Reply:
Theoretical yes, however the there is an issue in SHAP when you pass model like KNN or SVM(kernel='rbf') it throws an error. There is an open [issue about it](https://github.com/slundberg/shap/issues/1526) . We can mention now what models work and later update it.

Reply:
Okay, then lets add a todo to include it once SHAP updates the version again

Reply:
Will create an issue about it.